cypress/common/base.ts

```diff
@@ -749,11 +751,20 @@ export class BaseMethods {
         cy.skipWhen(condition)
     }
 
-    public openLocalhost(number: number, path?: string): Cypress.Chainable<Cypress.AUTWindow> {
+    public openLocalhost({
+        number,
+        path,
+        wait=0
+    }: {
+        number: number,
+        path?: string,
+        wait?: number
+    }): Cypress.Chainable<Cypress.AUTWindow> {
         return path ?
-            cy.visit(`${Cypress.env(`localhost${number}`)}/${path}`)
+            cy.visit(`${Cypress.env(`localhost${number}`)}/${path}`).wait(wait)
```

Comment:
do not like the idea of wait with visit. Try use waitUntil with param something like "isWithWait" or just move wait to spec level